test/test_web.rb

```diff
@@ -663,4 +663,46 @@ def app
       assert_equal 'nicehost.org', session_options[:host]
     end
   end
+
+  describe 'sidekiq web sessions options' do
+    include Rack::Test::Methods
+
+    def app
+      Sidekiq::Web.new
+    end
+
+    after { Sidekiq::Web.enable(:sessions) }
```

Comment:
@mperham Is up to you if you want to introduce this change, my initial design prioritized retrocompatibility so that users of the old Sinatra API would'nt suffer with the migration.